frontend/src/components/header/index.js

```diff
@@ -21,17 +22,46 @@ const menuItems = [
   {label: messages.help, link: "help"}
 ];
 
+
+const ActiveNavLink = (props) => {
```

Comment:
```suggestion
const TopNavLink = (props) => {
```
if you agree, see below